website/docs/d/workspace.html.markdown

```diff
@@ -49,6 +49,7 @@ In addition to all arguments above, the following attributes are exported:
 * `speculative_enabled` - Indicates whether this workspace allows speculative plans.
 * `ssh_key_id` - The ID of an SSH key assigned to the workspace.
 * `structured_run_output_enabled` - Indicates whether runs in this workspace use the enhanced apply UI. 
+* `tag_names` - The names of tags added to this workspace.
```

Comment:
We'll need to change the documentation above in the _Argument Reference_ to say that either `name` or `tag_names` are required, but at least one is.